templates/navigation/navigation-nojs.html

```diff
@@ -0,0 +1,158 @@
+{% set nav = get_navigation("all") %}
+
+{% extends "base_index.html" %}
+
+{% block title %}Navigation{% endblock %}
+
+{# Macro for rendering three-column link layout #}
+
+{% macro render_three_column_links(links, has_secondary_links=false) %}
+  {% set columns = 3 if not has_secondary_links else 2 %}
+  {% set column_links = split_list(links, columns) %}
+
+  {% for i in range(columns) %}
+    <div class="col-3">
```

Comment:
This would be a good opportunity to migrate to the latest grid pattern. If you feel this is out of scope we can create another maintenance issue for it.